tools/generator/templates/player/c/interface.cc.jinja2

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (c) 2020 Association Prologin <association@prologin.org>
+
+// This file is generated. It contains the code to call the API functions from
+// the C++ language. Do not edit this file.
+
+#include <vector>
+#include <string>
+
+extern "C" {
+#include "api.h"
+}
+
+template<typename Lang, typename Cxx>
+Cxx lang2cxx(Lang in)
```

Comment:
I don’t really like the name of this function. I find using `lang` instead of `c` harder to read. Same for `Lang`.

Also it would be nice to run clang-format on the template.